public/src/components/App.jsx

```diff
@@ -1,28 +0,0 @@
-// This component handles the App template used on every page.
-import React,{PropTypes} from 'react';
-import Header from './templates/header.jsx';
-import {connect} from 'react-redux';
-class App extends React.Component {
-  render() {
-    return (
-      <div>
-          <Header
-            logedIn={this.props.userLoged}
```

Comment:
misspelling, as long as it's consistent site-wide it's ok, but should be corrected if it will be used for demo purposes later.